### PR TITLE
Python snake (Workshop bounty)

### PR DESCRIPTION
**Pull request by kyryloren**
Followed all the workshop guidelines.

The club meeting showcase video: https://hackclub.slack.com/archives/C01A9V5LPD0/p1606605399044200

### CODE REVIEW

File: workshops/python_snake/README.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+Snake is a game that most of us have played on those old Nokia phones. It's also a classic coding challenge for learning a new programming language. As complicated as it may seem at first, it's rather easy to code and takes less than 50 lines. Let's try to recreate this game in the terminal.
+
+![Snake game demo in repl.it](https://cloud-h8v6zt88z.vercel.app/0snakereplit.gif)
+
+## Get started
+We are going to use [repl.it](https://repl.it) for this project. Just go to https://repl.it/languages/python3 to start coding. It's that easy! Creating an account will ensure you don't lose your code but you can do it after we finish this project. Let's start coding!
+
+#### Declare dependencies
+Today, we will be using `random` and `curses`. These should be at the top of the python file. These libraries are just python code that someone wrote.
```
Comment (faisalsayed10) [inline workshops/python_snake/README.md:16]:
```suggestion
Today, we will be using the `random` and `curses` libraries. These should be imported at the top of the python file. These libraries are just python code that someone else wrote for us to use.
```

File: workshops/python_snake/README.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    [snk_y, snk_x],
+    [snk_y, snk_x - 1],
+    [snk_y, snk_x - 2]
+]
+
+# Set the first food item at the center of the screen
+food = [sh/2, sw/2]
+```
+
+Think of the snake in blocks. The first block is the head. The head is at the initial `snk_x` and `snk_y` positions. This means that the the next body part has to be at 1 less X position than the head. The next body part is 2 less X from the head. The next 3. Each list item in the `snake` list is an initial body part position.
```
Comment (faisalsayed10) [inline workshops/python_snake/README.md:66]:
```suggestion
Think of the snake as a group of blocks. The first block is the head. The head is at the initial `snk_x` and `snk_y` positions. This means that the the next body part has to be at 1 less X position than the head. The next body part is 2 less X from the head. The next 3 and so on. Each list item in the `snake` list is an initial body part position.
```

File: workshops/python_snake/README.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
+---
+name: Snake Game
+description: The classic snake game recreated in Python
+author: '@kyryloren',
+img: 'https://cloud-i254770eq.vercel.app/0snake.png'
+---
+
+Snake is a game that most of us have played on those old Nokia phones. It's also a classic coding challenge for learning a new programming language. As complicated as it may seem at first, it's rather easy to code and takes less than 50 lines. Let's try to recreate this game in the terminal.
+
+![Snake game demo in repl.it](https://cloud-h8v6zt88z.vercel.app/0snakereplit.gif)
```
Comment (faisalsayed10) [inline workshops/python_snake/README.md:10]: We need to have the source code link and the final product link here

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
Hey! Awesome workshop! Just a few things which I noticed:

- Some functions from the curses library are a bit confusing, can u please explain each of them a bit more?

- This workshop lacks images and gifs, so at a point, the reader might just be looking at a wall of text and may get bored, I suggest you to add some gifs and images such as "this is what our preview should look like after we write this code", "this is how the snake looks like" "this is what it will look like when the game gets over"

**Comment by faisalsayed10** [comment]
Alright, I'll do a rereview tonight!

**Comment by faisalsayed10** [comment]
alright looks good woohoo! :tada:

**Review by faisalsayed10 — APPROVED** [review]

**Review by sampoder — CHANGES_REQUESTED** [review]
This is sick! Haven't really reviewed it but did a quick demo and was wondering if we could make the error message more user friendly: 

Screenshot 2020-12-24 at 8 41 38 AM

**Comment by kyryloren** [comment]
> This is sick! Haven't really reviewed it but did a quick demo and was wondering if we could make the error message more user friendly:
> 
> Screenshot 2020-12-24 at 8 41 38 AM

The program is supposed to be more of a quick and lightweight solution for a game in python. But yeah I can add it.

**Comment by MatthewStanciu** [comment]
Wow! This is fantastic! Well done 🎉

**Pull request merged**